conf/distro/bitsy.conf

```diff
@@ -29,39 +29,15 @@ MAINTAINER = "Leigh Johnson <leigh@bitsy.ai>"
 TARGET_VENDOR = "-bitsy"
 LOCALCONF_VERSION = "2"
 
-BITSY_DEFAULT_DISTRO_FEATURES = "\
-    acl \
-    argp \
-    bitsy-growpart \
-    bluetooth \
-    ext4 \
-    ipv4 \
-    ipv6 \
-    keyboard \
-    largefile \
-    overlayfs \
-    nginx \
-    seccomp \
-    swupdate \
-    systemd \
-    usbgadget \
-    usbhost \
-    vfat \
-    wifi \
-    xattrs \
-    zeroconf \
-"
+BITSY_DEFAULT_DISTRO_FEATURES = "acl argp bitsy-growpart overlayfs bluetooth ext4 ipv4 ipv6 keyboard largefile overlayfs nginx seccomp swupdate systemd usbgadget usbhost vfat wifi xattrs zeroconf"
```

Comment:
Wasn't sure if the formatting was impacting filter for `tensorflow-lite` - but looks like no